README.md

```diff
@@ -143,6 +144,20 @@ component.
 #### Inheritance
 
 The properties of the [`Box`](#box) component are also available.
+### Avatar
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Avatar/story.js)
+
+#### Props
+
+| Name   | Type   | Default   | Required   | Description   |
+| ------ | ------ | --------- | ---------- | ------------- |
+| `email`| `string`| - | - | The email to fetch the gravatar |
+| `fullName`| `string`| - | - | Full name will be splitted and will grab only the first letters  |
```

Comment:
This feels a bit hacky (what if my name is `First second third fourth`)? How about having first name and last name as separate props?

Reply:
TBH I had done exactly for that, it seemed comfortable to be able to pass a `fullName="First second third fourth"` and have as a result `FSTF`. Passing first and last name seems a little restrictive, but we can redo it in the future.

Reply:
@JSReds but the avatar won't fit 4 letters for example, and people normally have first name (multiple middle names) last name, so it is still valid.